### DISCUSSION TITLE
Добавить функцию getRelatedTracks

### DISCUSSION THREAD

**Discussion by Maracumba** [Chimildic/goofy#20] [Идея] [answered]
Исходя из функции **getRelatedArtists** появилась мысль. Есть ли такая возможность создать алгоритм, который будет собирать массив из рекомендованных Spotify треков на основе плейлиста или другого массива треков.
У Spotify есть такие треки внизу плейлистов, которые он предлагает добавить в ваш плейлист.

**Comment by Chimildic** [comment, marked as answer]
Есть [Lastfm.getSimilarTracks](https://chimildic.github.io/goofy/#/func?id=getsimilartracks) и [Source.getRecomTracks](https://chimildic.github.io/goofy/#/func?id=getrecomtracks). Сам Spotify других API-методов не дает.
